fee_calculator/apps/calculator/fixtures/scheme.json

```diff
@@ -28,5 +28,15 @@
     "base_type": 1,
     "description": "AGFS Fee Scheme 10"
   }
+},
+{
+  "model": "calculator.scheme",
+  "pk": 4,
+  "fields": {
+    "start_date": "2017-12-01",
+    "end_date": null,
+    "base_type": 2,
+    "description": "LGFS Fee Scheme 2017-12"
```

Comment:
Well, this was the problem - generally people have been referring to it as scheme 9, but according to Clem it's scheme 8 in CCLF, and now you're showing me a table saying it's scheme 10. I just wanted to avoid any ambiguity, basically.

Reply:
fair enough. since we will probably be determining fee scheme on `case_date` and `type` the description is not an issue in any event. maybe there should be `version` attribute on fee scheme in which this info could be returned. our CCCD current fee scheme table looks like this

```
  create_table "fee_schemes", force: :cascade do |t|
    t.string   "name"
    t.integer  "version"
    t.datetime "start_date"
    t.datetime "end_date"
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
  end
```

Reply:
github issue?